### PR TITLE
logrotate: don't exit at non-running instance

### LINKED ISSUE #774: `tt logrotate` does not work for started instances if others are stopped
Steps to reproduce:
```
$ tt status
INSTANCE       STATUS          PID
app1           NOT RUNNING     
app2:inst1     RUNNING         158686
app2:inst2     RUNNING         158687

$ tt logrotate
   ⨯ NOT RUNNING

$ tt logrotate app2
   • app2:inst1: logs has been rotated. PID: 158686.
   • app2:inst2: logs has been rotated. PID: 158687.
```

### PR DESCRIPTION
**Pull request by elhimov**
Just warn that the instance is not running and continue (like `tt stop` or `tt kill` do).

Closes #774

### CODE REVIEW

File: cli/cmd/logrotate.go [hunk 1]
```diff
@@ -47,11 +47,10 @@ func internalLogrotateModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 	}
 
 	for _, run := range runningCtx.Instances {
-		res, err := running.Logrotate(&run)
+		err := running.Logrotate(&run)
```
Comment (patapenka-alexey) [inline cli/cmd/logrotate.go:50]: It's better to use '_, err' to ignore the unused return value.

Comment (elhimov) [reply cli/cmd/logrotate.go:50]: It doesn't return `res` anymore.

Comment (patapenka-alexey) [reply cli/cmd/logrotate.go:50]: Ooops, my mistake.

**Review by patapenka-alexey — APPROVED** [review]
Thank you for the patch. LGTM.

**Review by dmyger — APPROVED** [review]

**Pull request merged**